tests/framework/utils_cpuid.py

```diff
@@ -26,10 +30,26 @@ def get_cpu_vendor():
 
 def get_cpu_model_name():
     """Return the CPU model name."""
-    _, stdout, _ = run_cmd("cat /proc/cpuinfo | grep 'model name' | uniq")
+    if platform.machine() == "aarch64":
+        _, stdout, _ = run_cmd("cat /proc/cpuinfo | grep 'CPU part' | uniq")
+    else:
+        _, stdout, _ = run_cmd("cat /proc/cpuinfo | grep 'model name' | uniq")
     info = stdout.strip().split(sep=":")
     assert len(info) == 2
-    return info[1].strip()
+    raw_cpu_model = info[1].strip()
+    if "Intel" in raw_cpu_model:
+        return raw_cpu_model
```

Comment:
Does this only apply for `Intel` or for `AMD` as well ?

Reply:
Can we simply test if the platform is `x86` ? Instead of testing for Intel and AMD.